version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "5.1.0-build.{height}",
+  "version": "5.1.1",
```

Comment:
We should keep the build.height syntax for daily builds. Not sure if we should use 5.2 or 6.0 as base for master yet... Hmmn

Reply:
```suggestion
  "version": "6.0.0-build.{height}",
```

Yeah, we should go to 6.0 at this point, or just revert this file from the branch change.  @PsychosisSquirm think you'll have time to make this small tweak and sign the CLA so we can get his PR merged?  Thanks!